### ISSUE TITLE
Accessibility - Projects page - button missing discernible text

### ISSUE THREAD

**Issue by terrimorton** [dotnet-foundation/website#862]
This issue was found by running the Accessibility Insights for Web browser extension in Chrome.

URL: https://dotnetfoundation.org/projects

![image](https://user-images.githubusercontent.com/4643903/118021667-7ac9ab00-b329-11eb-9dfc-8d0d5a083327.png)

Snippet: 
`<button id="search-projects" class="btn btn-primary mb-2"><i class="fa fa-search"></i></button>`

Fix ONE of the following:

- Element does not have inner text that is visible to screen readers
- aria-label attribute does not exist or is empty
- aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
- Element has no title attribute
- Element's default semantics were not overridden with role="none" or role="presentation"
